autosubmit/autosubmit.py

```diff
@@ -1166,20 +1165,21 @@ def _perform_deletion(experiment_path: Path, structure_db_path: Path, job_data_d
         return error_message
 
     @staticmethod
-    def copy_as_config(exp_id,copy_id):
-        for conf_file in os.listdir(os.path.join(BasicConfig.LOCAL_ROOT_DIR, copy_id,"conf")):
+    def copy_as_config(exp_id, copy_id):
+        for conf_file in os.listdir(os.path.join(BasicConfig.LOCAL_ROOT_DIR, copy_id, "conf")):
             # Copy only relevant files
             if conf_file.endswith((".conf", ".yml", ".yaml")):
                 shutil.copy(os.path.join(BasicConfig.LOCAL_ROOT_DIR, copy_id, "conf", conf_file),
-                            os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id, "conf", conf_file.replace(copy_id,exp_id)))
+                            os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id, "conf",
+                                         conf_file.replace(copy_id, exp_id)))
             # if ends with .conf convert it to AS4 yaml file
             if conf_file.endswith(".conf"):
                 try:
-                    AutosubmitConfig.ini_to_yaml(os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id,"conf"),
-                                                 os.path.join(BasicConfig.LOCAL_ROOT_DIR, exp_id,"conf",
-                                                              conf_file.replace(copy_id,exp_id)))
+                    AutosubmitConfig.ini_to_yaml(Path(BasicConfig.LOCAL_ROOT_DIR + "/" + exp_id + "/conf"),
```

Comment:
`Path(BasicConfig.LOCAL_ROOT_DIR, exp_id, 'conf')` the constructor of `Path` should act like `os.path.join`, so we don't need to worry if concatenating strings will produce a valid path, and the constructor of `Path` will do that for us :grimacing: